projects/dotcms/src/lib/models/DotCMSContentTypeField.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface DotCMSContentTypeField {
```

Comment:
Please move this one inside `DotCMSContentType.model.ts` what I'm trying is to have **one model** file per API, I think it would be cleaner that way.